src/plots/errorBarPlot.ts

```diff
@@ -0,0 +1,115 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Plots {
+  export class ErrorBar<X, Y> extends XYPlot<X, Y> {
+
+    private _tickLength = 10;
+
+    constructor() {
+      super();
+      this.addClass("error-bar-plot");
+    }
+
+    /**
+     * Sets the tick length
+     */
+    public tickLength(): number;
+    public tickLength(tickLength: number): ErrorBar<X, Y>;
+    public tickLength(tickLength?: number): any {
```

Comment:
Can we query to see the usecase of these `tickLength()` endpoints?  We can add `topTickLength()`/`bottomTickLength()` endpoints instead of / after this one and I think it should work nicely additively, but we should just choose the one more aligned with our usecase for now.